pkg/driver/node.go

```diff
@@ -188,13 +188,26 @@ func (d *nodeService) NodeStageVolume(ctx context.Context, req *csi.NodeStageVol
 		return &csi.NodeStageVolumeResponse{}, nil
 	}
 
+	existingFormat, err := d.mounter.GetDiskFormat(source)
+	if err != nil {
+		return nil, status.Error(codes.Internal, fmt.Sprintf("failed to get disk format of disk %s: %v", source, err))
+	}
```

Comment:
I guess `existingFormat` is never empty if `err` is nil ?

Reply:
The function throws an error if it can't determin the disk format (disk not found for example). But if the disk is not formatted, it return no error and existingFormat is empty (see [code](https://github.com/kubernetes/mount-utils/blob/6e81bcc03fc8c22aa460c3c4bd32a7ad602abd6c/mount_linux.go#L485))